Fix cannot choose the same profile image again after removing the existing one

Henry0422 opened the pull request:
For https://github.com/nanoPayinc/NANOPAY/issues/2236, I believe we need to set the `this.document.querySelector('.attachment-input').value ` to null if we remove the profile images.

Henry0422 committed:
Fix cannot choose the same image after removal

kgrgreer commented on src/foam/nanos/auth/ProfilePictureView.js:
```diff
@@ -200,6 +200,7 @@ foam.CLASS({
     function onRemoveClicked (e) {
       this.dragActive = false;
       this.ProfilePictureImage= null;
+      this.document.querySelector('.attachment-input').value = null;
```
What if there's more than one attachment-input on the screen at once? I'll show you a better way to do this.

Henry0422 replied:
What would be the better solution for this issue? Thanks.

Henry0422 committed:
Merge pull request #3 from foam-framework/master

Update from foam2

Commit:
Modified Date refinements to use SimpleDateFormat to parse dates

Commit:
Merge branch 'master' of github.com:foam-framework/foam2 into ReplaceDeprecatedDateAPIs

Commit:
Removed unused import

kgrgreer committed:
Merge pull request #1385 from eatonk/ReplaceDeprecatedDateAPIs

Replace deprecated new Date(string) method calls

adamvy committed:
Jetty HttpServer add factories to avoid NPE

Add default factories to servlet/errorPageMapping properties to avoid NPE if they haven't been configured.

adamvy committed:
Comment out reference to nanopay specific view in UserView

Henry0422 committed:
Merge pull request #4 from foam-framework/master

Update from main foam2 repo

Henry0422 committed:
Merge branch 'master' of https://github.com/Henry0422/foam2 into ProfileImageReload

Henry0422 committed:
Update the fix for cannot choose the same image after removal
